src/Login/SingUp/LoginUsername.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
+import Translate from 'react-i18nify/build/lib/Translate';
```

Comment:
@CharlyJazz  Why don't you just add `import { Translate } from 'react-i18nify'`

Reply:
Oh, it was my mistake, my text editor automatically creates the route to import

I'll be more focused on avoiding those mistakes